---
layout: new
title: Publications
permalink: /publications/
---
\# Co-first authors; \* Co-corresponding authors

## Journal Articles

### 2024
1. Lei Wang#, Zilu Zeng#, Zhidong Xue\*, and Yan Wang\*. (2024). **DeepNeuropePred: A robust and universal tool to predict cleavage sites from neuropeptide precursors by protein language model**. Computational and Structural Biotechnology Journal, 23, 309–315. <https://doi.org/10.1016/j.csbj.2023.12.004>
2. Mingxia Wang#, Lei Wang#, Wei Xu, Ziqiang Chu, Hengzhi Wang, Jingxiang Lu, Zhidong Xue, and Yan Wang. (2024). **NeuroPep 2.0: An Updated Database Dedicated to Neuropeptide and Its Receptor Annotations**. Journal of Molecular Biology, 436(4), 168416. <https://doi.org/10.1016/j.jmb.2023.168416>

### 2023
1. Lei Wang, Chen Huang, Mingxia Wang, Zhidong Xue\*, and Yan Wang\*. (2023). **NeuroPred-PLM: An interpretable and robust model for neuropeptide prediction by protein language model**. Briefings in Bioinformatics, 24(2), bbad077. <https://doi.org/10.1093/bib/bbad077>
2. Lei Wang, Hui Zhang, Wei Xu, Zhidong Xue\*, and Yan Wang\*. (2023). **Deciphering the protein landscape with ProtFlash, a lightweight language model**. Cell Reports Physical Science, 4(10). <https://doi.org/10.1016/j.xcrp.2023.101600>

### 2022
1. Lei Wang, Haolin Zhong, Zhidong Xue\*, and Yan Wang\*. (2022). **Improving the topology prediction of α-helical transmembrane proteins with deep transfer learning**. Computational and Structural Biotechnology Journal, 20, 1993–2000. <https://doi.org/10.1016/j.csbj.2022.04.024>
2. Lei Wang, Haolin Zhong, Zhidong Xue\*, and Yan Wang\*. (2022). **Res-Dom: Predicting protein domain boundary from sequence using deep residual network and Bi-LSTM**. Bioinformatics Advances, 2(1), vbac060. <https://doi.org/10.1093/bioadv/vbac060>

### 2021
1. Yan Wang, Hang Zhang, Haolin Zhong, and Zhidong Xue. (2021). **Protein domain identification methods and online resources**. Computational and Structural Biotechnology Journal, 19, 1145–1153. <https://doi.org/10.1016/j.csbj.2021.01.041>

### 2020
1. Weiya Chen, Chenchen Yu, Chenyu Tu, Zehua Lyu, Jing Tang, Shiqi Ou\*, Yan Fu\*, and Zhidong Xue\*. (2020). **A Survey on Hand Pose Estimation with Wearable Sensors and Computer-Vision-Based Methods**. Sensors, 20(4), Article 4. <https://doi.org/10.3390/s20041074>
2. Weiya Chen, Chun Yao, Yingzhong Guo, Yan Wang, and Zhidong Xue. (2020). **pmTM-align: Scalable pairwise and multiple structure alignment with Apache Spark and OpenMP**. BMC Bioinformatics, 21(1), 426. <https://doi.org/10.1186/s12859-020-03757-2>

### 2019
1. Qiang Shi, Weiya Chen, Siqi Huang, Yan Wang\*, and Zhidong Xue\*. (2019). **Deep learning for mining the protein data**. Briefings in Bioinformatics, 22(1), 194–218. <https://doi.org/10.1093/bib/bbz156>
2. Yan Wang#, Qiang Shi#, Pengshuo Yang#, Chengxin Zhang#, S. M. Mortuza, Zhidong Xue\*, Kang Ning\*,and Yang Zhang\*. (2019). **Fueling ab initio folding with marine metagenomics enables structure and function predictions of new protein families**. Genome Biology, 20(1), 229. <https://doi.org/10.1186/s13059-019-1823-z>
3. Qiang Shi, Weiya Chen, Siqi Huang, Fanglin Jin, Yinghao Dong, Yan Wang\*, and Zhidong Xue\*. (2019). **DNN-Dom: predicting protein domain boundary from sequence alone by deep neural network**. Bioinformatics, 35(24), 5128–5136. <https://doi.org/10.1093/bioinformatics/btz464>

### 2018
1. Qiang Shi#, Weiya Chen#, Ye Pan, Shan Yin, Yan Fu, Jiacai Mei\*, and Zhidong Xue*.(2018). **An Automatic Classification Method on Chronic Venous InSufficiency Images**. Scientific Reports, 8(1), 17952. <https://doi.org/10.1038/s41598-018-36284-5>
2. Jian Wang#, Tailang Yin#, Xuwen Xiao, Dan He, Zhidong Xue, Xinnong Jiang\*, and Yan Wang\*. (2018). **StraPep: a Structure Database of Bioactive Peptides**. Database, 2018, bay038. <https://doi.org/10.1093/database/bay038>

### 2017
1. Yan Wang, Jian Wang, Ruiming Li, Qiang Shi, Zhidong Xue\*, and Yang Zhang\*. (2017). **ThreaDomEx: a unified platform for predicting continuous and discontinuous protein domains by multiple-threading and segment assembly**. Nucleic Acids Research, 45(W1), W400–W407. <https://doi.org/10.1093/nar/gkx410>
2. Yan Wang, Jouko Virtanen, Zhidong Xue\*, and Yang Zhang\*. (2017). **I-TASSER-MR: automated molecular replacement for distant-homology proteins using iterative fragment assembly and progressive sequence truncation**. Nucleic Acids Research, 45(W1), W429–W434. <https://doi.org/10.1093/nar/gkx349>

### 2016
1. Yan Wang, Jouko Virtanen, Zhidong Xue, John J. G. Tesmer, and Yang Zhang. (2016). **Using iterative fragment assembly and progressive sequence truncation to facilitate phasing and crystal structure determination of distantly related proteins**. Acta Crystallographica Section D: Structural Biology, 72(5), 616–628. <https://doi.org/10.1107/S2059798316003016>
2. Jianyi Yang, Yan Wang, and Yang Zhang\*. (2016). **ResQ: An Approach to Unified Estimation of B-Factor and Residue-Specific Error in Protein Structure Prediction**. Journal of Molecular Biology, 428(4), 693–701. <https://doi.org/10.1016/j.jmb.2015.09.024>

### 2015
1. Zhidong Xue*, Richard Jang, Brandon Govindarajoo, Yichu Huang, and Yan Wang\*. (2015). **Extending Protein Domain Boundary Predictors to Detect Discontinuous Domains**. PLOS ONE, 10(10), e0141541. <https://doi.org/10.1371/journal.pone.0141541>








