---
layout: new
title: Software
permalink: /software/
---

## Bioinformatics tools and database

- **[ThreaDomEx](http://isyslab.info/ThreaDomEx/)**  
  Protein domains are subunits that can fold and evolve independently. The identification of protein domains is essential for protein structure determination and functional annotations. ThreaDom2 and DomEx3 are two methods recently developed for protein domain boundary recognition and especially discontinuous domain prediction. ThreaDomEx combines ThreaDom and DomEx into a unified on-line server system for more accurate and user-friendly domain predictions on sequences of both continuous and discontinuous domain structures. ThreaDomEx takes the amino acid sequence of the query protein as input. It first creates multiple threading alignments to recognize homologous and analogous template structures, from which a domain conservation score is then calculated for deducing the domain boundaries. Next, a boundary clustering method is used to optimize the domain model selections. For discontinuous domain structures, a symmetric alignment algorithm is applied to further integrate and refine the domain assignments. Output of the server consists of: (a) the predicted domain boundaries and discontinuous domains; (b) the visualized distribution of domain conserve score, predicted secondary structure and solvent accessiblity; (c) the threading templates used by ThreaDomEx. The server allows users to interactively edit, save, or re-detect the domain models of the proteins.  
- **[ThreaDom](https://zhanglab.ccmb.med.umich.edu/ThreaDom)**  
  ThreaDom is Protein Domain prediction tools based on several protein threading programs. It not only can report protein domain boundaries more accurately than most protein domain predictors especially for the medium and hard target, but also can detect the discontinuous domains based on boundary clustering.  
- **[DomEx](http://isyslab.info/DomEx/)**  
  DomEx enables domain boundary predictors to detect discontinuous domains by assembling the continuous domain segments. Discontinuous domains are predicted by matching the sequence profile of concatenated continuous domain segments with the profiles from a single-domain library derived from SCOP and CATH, and Pfam. Then the matches are filtered by similarity to library templates, a symmetric index score and a profile-profile alignment score. Compared with ThreaDom DomEx recalled 26.7% discontinuous domains with 72.7% precision in a benchmark with 29 discontinuous-domain chains, where ThreaDom failed to predict any discontinuous domains. The source code and datasets are available [here](http://isyslab.info/DomEx/).
- **[NeuroPep](http://isyslab.info/NeuroPep/)**  
  Neuropeptides are involved in a number of physiological processes and serve as potential therapeutic targets for the treatment of some nervous-system disorders. We have developed a comprehensive neuropeptide database, NeuroPep, which is currently the most complete neuropeptide database.
  The current release of the database (version 1.0, 2014-11-26) holds 5949 non-redundant neuropeptide entries originating from 493 organisms belonging to 65 neuropeptide families. The data were collected from various resources including MEDLINE abstracts, full papers, UniProtï¼Œdatabase at www.neuropeptides.nl and Neuropedia. All the entries in NeuroPep have been manually checked.  
- **[StraPep](https://github.com/ISYSLAB-HUST/StraPep)**  
  StraPep is a database dedicated to annotation of bioactive peptides with known structure, which often function as modulator in physiological processes by means of interacting with specific receptor. There are 1199 bioactive peptides and 3536 PDB chains in CBPS. All the peptides are classified into 6 categories: Toxin and Venom peptide (441), Antimicrobial peptide (365), Cytokine and Growth factor (199), Hormone (132), Neuropeptide (37) and Others (25). All the structure information of peptides were collected from Protein Data bank, and the sequence information were mainly from Uniprot. Users can easily access peptide interested by user-friendly browser and search engine. For each entry, the detail page provides both primary information and structure information. Tools like BLAST, Map and Secondary structure composition are also provided.  
- **[I-TASSER-MR](https://zhanglab.ccmb.med.umich.edu/I-TASSER-MR)**  
  I-TASSER-MR has been developed to test whether the success rate for structure determination of distant-homology proteins could be improved by a combination of iterative fragmental structure-assembly simulations with progressive sequence truncation designed to trim regions with high variation. The pipeline was tested on two independent protein sets consisting of 61 proteins from CASP8 and 100 high-resolution proteins from the PDB. After excluding homologous templates, I-TASSER generated full-length models with an average TM-score of 0.773, which is 12% higher than the best threading templates. Using these as search models, I-TASSER-MR found correct MR solutions for 95 of 161 targets as judged by having a TFZ of >8 or with the final structure closer to the native than the initial search models. The success rate was 16% higher than when using the best threading templates. I-TASSER-MR was also applied to 14 protein targets from structure genomics centers. Seven of these were successfully solved by I-TASSER-MR. These results confirm that advanced structure assembly and progressive structural editing can significantly improve the success rate of MR for targets with distant homology to proteins of known structure.  

For detailed introduction and online access to some tools, please visit [our GitHub page](https://github.com/ISYSLAB-HUST). 